app/models/registration.rb

```diff
@@ -1,11 +1,12 @@
 # frozen_string_literal: true
 
 class Registration < ApplicationRecord
-  scope :pending, -> { where(accepted_at: nil).where(deleted_at: nil).where(rejected_at: nil).where(waitlisted_at: nil).where(is_competing: true) }
+  scope :pending, -> { where(accepted_at: nil, deleted_at: nil, is_competing: true) }
   scope :accepted, -> { where.not(accepted_at: nil).where(deleted_at: nil) }
   scope :deleted, -> { where.not(deleted_at: nil) }
-  scope :rejected, -> { where.not(rejected_at: nil) }
-  scope :waitlisted, -> { where.not(waitlisted_at: nil) }
+  scope :cancelled, -> { where(competing_status: 'cancelled') }
+  scope :rejected, -> { where(competing_status: 'rejected') }
+  scope :waitlisted, -> { where(competing_status: 'waiting_list') }
```

Comment:
Defining an `enum` in Rails gives you scopes (that are named like the keys of the hash) for free!

Reply:
yes, I use that in some of the methods, was just using following the convention of the other two scopes. Should we migrate those away in v3 cleanup and just use the built in enum scopes everywhere?